servers/rendering/renderer_rd/forward_mobile/scene_shader_forward_mobile.cpp

```diff
@@ -162,11 +162,11 @@ void SceneShaderForwardMobile::ShaderData::set_code(const String &p_code) {
 
 #ifdef DEBUG_ENABLED
 	if (uses_sss) {
-		WARN_PRINT_ONCE_ED("Sub-surface scattering is only available when using the Forward+ rendering backend.");
+		WARN_PRINT_ONCE_ED("Sub-surface scattering is only available when using the Forward+ renderer.");
```

Comment:
Another uncommon hyphen.
```suggestion
		WARN_PRINT_ONCE_ED("Subsurface scattering is only available when using the Forward+ renderer.");
```